### PR TITLE
Pass nextMonth to prev and next month click props

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

File: src/components/DayPicker.jsx [hunk 1]
```diff
@@ -629,14 +618,14 @@ class DayPicker extends React.Component {
     const newMonth = currentMonth.clone();
     const firstDayOfWeek = this.getFirstDayOfWeek();
     if (monthTransition === PREV_TRANSITION) {
-      if (onPrevMonthClick) onPrevMonthClick();
       newMonth.subtract(1, 'month');
+      if (onPrevMonthClick) onPrevMonthClick(newMonth);
```
Comment (majapw) [inline src/components/DayPicker.jsx]: This is exactly what I was going to suggest. :)

Comment (mmahalwy) [reply src/components/DayPicker.jsx]: haha nice!

**Comment by mmahalwy** [comment]
ok, removed the prettier changes!

**Comment by mmahalwy** [comment]
good to merge?

**Pull request merged**